io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/CommonSchemaConstants.java

```diff
@@ -25,7 +25,7 @@
 	/**
 	 * the parameter specifying the reader setting
 	 */
-	public static final String PARAM_SKIP_N_LINES = "skip";
+	public static final String PARAM_SKIP_N_LINES = "skipN";
```

Comment:
Regarding the behavior you chose now to replace the existing parameter and be backwards compatible on it. So the name needs to stay `skip`.